Client/Experiments/Messaging/GleanPlumbMessageUtility.swift

```diff
@@ -17,15 +17,15 @@ class GleanPlumbMessageUtility {
 
     // MARK: - Public helpers
 
-    func createGleanPlumbHelper() -> GleanPlumbMessageHelper? {
+    func createGleanPlumbHelper() -> NimbusMessagingHelperProtocol? {
```

Comment:
cc @jhugman @adudenamedruby 

Encountered those breaking changes in the latest nightly. Propagated the new type where I could to get things to build and tests to pass but wanted to explicitly flag you